static/elements/chromedash-app.js

```diff
@@ -100,22 +100,26 @@ class ChromedashApp extends LitElement {
     page('/', () => page.redirect('/roadmap'));
     page('/roadmap', (ctx) => {
       this.pageComponent = document.createElement('chromedash-roadmap-page');
+      this.pageComponent.user = this.user;
```

Comment:
Sometimes user permissions do change, but I think that just loading the same user permissions once is OK because: (a) the server checks updated permissions on POSTs, and (b) users who request new permissions can be expected to reload.